data/schema/v2/Decision_Point_Value_Selection-2-0-0.schema.json

```diff
@@ -0,0 +1,226 @@
+          "GHSA-0123-4567-89ab"
+        ]
+      ],
+      "items": {
+        "type": "string"
+      },
+      "minItems": 1,
+      "title": "Target Ids",
+      "type": "array"
+    },
+    "selections": {
+      "description": "List of selections made from decision points. Each selection item corresponds to value keys contained in a specific decision point identified by its namespace, key, and version. Note that selection objects are deliberately minimal objects and do not contain the full decision point details.",
+      "items": {
+        "$ref": "#/$defs/Selection"
+      },
+      "minItems": 1,
+      "title": "Selections",
+      "type": "array"
+    },
+    "resources": {
+      "description": "A list of references to resources that provide additional context about the decision points found in this selection.",
+      "examples": [
+        [
+          {
+            "description": "Documentation for a set of decision points",
+            "uri": "https://example.com/decision_points"
+          },
+          {
+            "description": "JSON representation of decision point 2",
+            "uri": "https://example.org/definitions/dp2.json"
+          },
+          {
+            "description": "A JSON file containing extension decision points in the x_com.example namespace",
+            "uri": "https://example.com/ssvc/x_com.example/decision_points.json"
+          }
+        ]
+      ],
+      "items": {
+        "$ref": "#/$defs/Reference"
+      },
+      "minItems": 1,
+      "title": "Resources",
+      "type": "array"
+    },
+    "references": {
+      "description": "A list of references to resources that provide additional context about the specific values selected.",
+      "examples": [
+        [
+          {
+            "description": "A report on which the selections were based",
+            "uri": "https://example.com/report"
+          }
+        ]
+      ],
+      "items": {
+        "$ref": "#/$defs/Reference"
+      },
+      "minItems": 1,
+      "title": "References",
+      "type": "array"
+    }
+  },
+  "required": [
+    "timestamp",
+    "schemaVersion",
+    "selections"
+  ],
+  "additionalProperties": false,
+  "$defs": {
+    "MinimalDecisionPointValue": {
+      "description": "A minimal representation of a decision point value.\nIntended to parallel the DecisionPointValue object, but with fewer required fields.\nA decision point value is uniquely identified within a decision point by its key.\nGlobally, the combination of Decision Point namespace, key, and version coupled with the value key\nuniquely identifies a value across all decision points and values.\nOther required fields in the DecisionPointValue object, such as name and description, are optional here.",
+      "properties": {
+        "key": {
+          "title": "Key",
+          "type": "string"
+        },
+        "name": {
+          "title": "Name",
+          "type": "string"
+        },
+        "description": {
```

Comment:
`description` is also a JSON keyword. Can we replace that to avoid confusion for implementers?

Reply:
Not without changing a lot of existing python code and documentation text. If this is a big deal, we'll need to prioritize it in a separate issue.